src/sentry/similarity/__init__.py

```diff
@@ -19,14 +18,15 @@
 )
 from sentry.similarity.signatures import MinHashSignatureBuilder
 from sentry.utils import redis
+from sentry.utils.compat import map
 from sentry.utils.datastructures import BidirectionalMapping
 from sentry.utils.iterators import shingle
 
 logger = logging.getLogger(__name__)
 
 
 def text_shingle(n, value):
-    return itertools.imap(u"".join, shingle(n, value))
+    return map(u"".join, shingle(n, value))
```

Comment:
Is this right? Means we'll have an iterator in py3 and a dict in py2.

Reply:
No iterator, it'll be a list because of `from sentry.utils.compat import map`.

Wdym dict in py2?

Reply:
Derp, I meant list but my brain went `a map is a dict`